atom/browser/api/lib/web-contents.js

```diff
@@ -106,16 +112,37 @@ let wrapWebContents = function(webContents) {
     };
   }
 
+  for (let method of asyncWebFrameMethods) {
+    webContents[method] = function() {
```

Comment:
It is not needed to add `asyncWebFrameMethods` to webContents, since there is only one method and it is overridden immediately in following code.